api_tests/nodes/views/test_node_contributors_list.py

```diff
@@ -1036,8 +1036,7 @@ def test_add_unregistered_contributor_already_contributor(
             auth=user.auth, expect_errors=True)
         project_public.reload()
         assert res.status_code == 400
-        assert res.json['errors'][0]['detail'] == '{} is already a contributor.'.format(
-            name)
+        assert res.json['errors'][0]['detail'] == 'Osf user with this Username already exists.'
```

Comment:
Not sure where this error detail is coming from, but we generally avoid confirming that an account exists in our DB for security reasons.